addons/web/static/src/core/l10n/dates.js

```diff
@@ -349,7 +349,10 @@ export function formatDateTime(value, options = {}) {
     if (!value) {
         return "";
     }
-    const format = options.format || localization.dateTimeFormat;
+    var format = options.format || localization.dateTimeFormat;
+    if (!(options.showSeconds ?? true)) {
+        format = format.replace(/:ss/, '');
```

Comment:
only works for formats where `:` is used as hours/minutes/seconds separator

Reply:
(we're trying to find the correct solution)

Reply:
@laro-odoo here's the time format without seconds https://github.com/odoo/odoo/pull/172941

Reply:
@laro-odoo https://github.com/odoo/odoo/pull/172941 has been merged, could you adapt your code to use it?

Reply:
Sure, I was also waiting for the PR to be merged.